<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>491</width>
     <height>141</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <column>
    <property name="text">
     <string>设备ID</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>设备名称</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>产品名称</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>在线状态</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>设备描述</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>设备列表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>查询设备</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="query_device_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_query_device_id">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>输入设备ID进行查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_device_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 10px;
background-color: #dfdfdf;
}

QPushButton:hover {
background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>开始查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>更新设备</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="update_device_id_input">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>输入要更新的设备ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="update_device_name_input">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>输入设备新的名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="update_device_desc_input">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>输入设备描述信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_device_button">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 10px;
background-color: #dfdfdf;
}

QPushButton:hover {
background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>更新设备</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>井盖设备数据</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="temp_lcd">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber {background-color: #333; color: white; border-radius: 10px;}</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="accel_x_lcd">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber {background-color: #333; color: white; border-radius: 10px;}</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="accel_y_lcd">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>360</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber {background-color: #333; color: white; border-radius: 10px;}</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="accel_z_lcd">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>360</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber {background-color: #333; color: white; border-radius: 10px;}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>330</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Accel_X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>330</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Accel_Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>330</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Accel_Y</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="smoke_value_lcd">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber {background-color:#8d8d8d; color: white; border-radius: 10px;}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>510</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Beep_Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>510</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoke_Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>460</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>烟感设备数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="beep_status_text">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>540</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cover_status</string>
   </property>
  </widget>
  <widget class="QLabel" name="cover_status_text">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>烟感设备状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>井盖设备状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="smoke_state_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="cover_state_text">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>500</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>下发井盖指令</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cover_command_input">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>50</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>对应指令 Open_Alert，值 ON,OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cover_command_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 10px;
background-color: #dfdfdf;
}

QPushButton:hover {
background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>发起指令</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="smoke_command_input">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>180</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>对应指令 Open_Alert，值 ON,OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="smoke_command_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 10px;
background-color: #dfdfdf;
}

QPushButton:hover {
background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>发起指令</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>下发烟感指令</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
